### PR TITLE
feat: added validation on org name

### LINKED ISSUE #1308: Add validation check for org name to be 20 characters or less
### Context
Now that we are revamping the org home page, we will need to set limits to org names so that it fits nicely within our designs. 

Screenshot 2024-01-09 at 12 30 45 PM

Let's limit the width to the name of an organization to 20 characters.

### Acceptance Criteria
- [ ] I've added a validation check that the length of an org name is 20 characters or less. If it greater than 20 then a response will be returned as "Error: exceeds 20 character limit".
- [ ] Create a unit test that checks if validation fails and passes

### PR DESCRIPTION
**Pull request by wright-eric**
## Describe your changes
added validation of 20 character on org name

## Issue ticket number and link
close #1308 

## Type of change
Feature

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Checklist before requesting a review
- [x] I have performed a self-review of my code

## Added changes
 - [x] I've added a validation check that the length of an org name is 20 characters or less. If it greater than 20 then a response will be returned as "Error: organization name must be present and should not exceed 20 character".
- [x] Create a unit test that checks if validation fails and passes

### CODE REVIEW

File: handlers/organizations.go [hunk 1]
```diff
@@ -44,6 +44,13 @@ func CreateOrEditOrganization(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if len(org.Name) == 0 || len(org.Name) > 20 {
+		fmt.Printf("invalid organization name %s\n", org.Name)
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode("Error: organization name must be present and should not exceed 20 character")
```
Comment (wright-eric) [inline handlers/organizations.go:50]: @ecurrencyhodler  i have changed the error message from what was mentioned in the issue to make more sense. let me know if you want me to revert back to the once mentioned in the issue

**Comment by wright-eric** [comment]
cc @kevkevinpal @elraphty

**Comment by kevkevinpal** [comment]
this looks good to me @elraphty can you take a look?

Also @ecurrencyhodler can we bump the bounty @wright-eric added extra tests checking that the [pubkey isn't empty](https://github.com/stakwork/sphinx-tribes/pull/1328/files#diff-1e998e2959e2c08e32625512775fb0b52025c1d93af579a193292cc70a74aa4eR36), [Body is not valid json](https://github.com/stakwork/sphinx-tribes/pull/1328/files#diff-1e998e2959e2c08e32625512775fb0b52025c1d93af579a193292cc70a74aa4eR21) and [org name is empty](https://github.com/stakwork/sphinx-tribes/pull/1328/files#diff-1e998e2959e2c08e32625512775fb0b52025c1d93af579a193292cc70a74aa4eR51)

**Comment by wright-eric** [comment]
> @wright-eric if you can also bump the amount in `sphinx-tribes/.github/workflows/.prjob_tests.yml` that would be great!

sure will do that

**Comment by ecurrencyhodler** [comment]
Hey eric. Kev mentioned you added extra tests. We really appreciate that. We're going to bump your bounty an extra 50k sats.

**Comment by ecurrencyhodler** [comment]
Paid
